src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -13,16 +13,27 @@ const Authenticate = (() => {
             if (response.error) {
                 $('#error_message').setVisibility(1).text(response.error.message);
             } else {
-                const status = response.get_account_status.status;
-                if (!/authenticated/.test(status)) {
-                    init();
-                    const $not_authenticated = $('#not_authenticated').setVisibility(1);
-                    let link = 'https://marketing.binary.com/authentication/2017_Authentication_Process.pdf';
-                    if (Client.isAccountOfType('financial')) {
-                        $('#not_authenticated_financial').setVisibility(1);
-                        link = 'https://marketing.binary.com/authentication/2017_MF_Authentication_Process.pdf';
+                const get_account_status  = response.get_account_status;
+                const should_authenticate = +get_account_status.prompt_client_to_authenticate;
```

Comment:
In order to allow unauthenticated clients to upload their documents (i.e. for MT5 withdrawal), we are [not checking this flag](https://github.com/binary-com/binary-static/pull/3417/commits/f76609175865c5fd21d8d21572b28bd678d442d5#diff-fba57d9cce18f8e420f158772b4557e5L17) here anymore.